src/test/articlesTest.js

```diff
@@ -1,12 +1,24 @@
 import chai from 'chai';
 import chaiHttp from 'chai-http';
 import app from '../index';
+import models from '../models';
+import Tokenizer from '../helpers/tokenGenerator';
+
+const { User, Articles } = models;
+const { generateToken } = Tokenizer;
 
 chai.use(chaiHttp);
 chai.should();
 
+
 let tokens = ' ';
 const tokens1 = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJlbWFpbCI6ImZhdXN0aW5rYWdhYm8xMUBnbWFpbC5jb20iLCJpYXQiOjE1NjMxMDE5NTYsImV4cCI6MTU2MzI3NDc1Nn0.33LK5oq_7jD-aUbAvY6XTeWNZ-GgaFdibNivPNKQgvA';
+let userObject;
+let articleObject;
+let testUser;
+let testArticle;
+let tokenGen;
+
 
 describe('Article', () => {
   it('A new user who filled all required data should be registered in order to create an article', (done) => {
```

Comment:
We can extract this before script into another file to avoid redundancy because it is being used in more that one file .